accounts/templates/account/login.html

```diff
@@ -0,0 +1,32 @@
+{% extends "exp/base.html" %}
+
+{% load i18n %}
+{% load account socialaccount %}
+{% load bootstrap3 %}
+
+{% block head_title %}{% trans "Sign In" %}{% endblock %}
```

Comment:
What are we going to do eventually about this secret login page? First, I think experimenter should have its own landing page instead of defaulting to Manage Studies (since this page has specific permissions). 

Then, Experimenter pages have an experimenter login button which takes them to experimenter login (through oauth), Lookit pages have Lookit login button. If you're logged out on experimenter page you get experimenter login. Logged out on Lookit page get Lookit login.

If researcher logs in first time through oauth, he is redirected to new Experimenter landing page instead of leaving him in lookit land. Thoughts?